### DISCUSSION TITLE
Does a Notification CC "Manual Lock/Unlock Operation" mean the device is (un)locked?

### DISCUSSION THREAD

**Discussion by AlCalzone** [zwave-js/zwave-js#1601] [General] [answered]
@hanskroner This could be a short one. I'm getting reports of users who have locks that report a manual change in lock state via the Notification CC instead of (Door) Lock CC. 
Does it make sense to assume that those are equivalent, e.g. Notification CC "Manual Lock Operation" = Door Lock CC Report "closed"?

**Comment by hanskroner** [comment, marked as answer]
It makes sense, but they're not exactly equivalent. You'll need to deal with this when the S2 implementation lands.

The Door Lock Command Class is constrained to the _S2 Access Control_ key, while the Notification Command Class, in the same device, could use a different key. This way the device can notify devices which don't belong to the _S2 Access Control_ key group of changes in its state.
